src/routes/JoinGroup/JoinGroup.js

```diff
@@ -56,7 +55,8 @@ export default function JoinGroup (props) {
           }
         }
       } catch (error) {
-        history.goBack()
+        window.alert('Sorry, your invitation to this group is expired, has already been used, or invalid. Please contact a group moderator for another one.')
```

Comment:
"or is invalid" (missing an is)

Reply:
Grabbed it from elsewhere in the app, I'll update both ;)